app/main.py

```diff
@@ -25,30 +25,36 @@
 from slowapi import _rate_limit_exceeded_handler
 from slowapi.errors import RateLimitExceeded
 
+from .auth import authlib_oauth
 from .controllers import auth_router, example_router, health_router
 from .settings import api_settings, rate_limiter
 
 fastAPI_logger = logger  # convenient name
 
 
 def get_application() -> FastAPI:
-    tags_metadata = [
-        {
-            "name": "apikeymanager",
-            "description": "Operations with users. The **login** logic is also here.",
-        },
-    ]
+    # tags_metadata = [
+    #     {
+    #         "name": "apikeymanager",
+    #         "description": "Operations with users. The **login** logic is also here.",
+    #     },
+    # ]
+    tags_metadata = None
 
     application = FastAPI(
         title=api_settings.name,
-        description="APIKeyManager is a centralized Python-oriented API Key manager.",
+        description=api_settings.swagger_description,
```

Comment:
Allow to customize the description displayed in the swagger front page. See the settings.py module.